src/protocols/pcie.stanza

```diff
@@ -70,22 +70,26 @@ public defn PCIe-enum-to-int (en:PCIeWidth) -> Int:
     PCIe-x16 : 16
     PCIe-x32 : 32
 
+public pcb-enum jsl/bundles/pcie/PCIePins :
+  PCIe-PRSNT#
+
 doc: \<DOC>
 @brief PCI-e Bundle
 The Data Bundle consists of a number (1 or more) of RX and TX pairs (1 RX/TX pair set = 1 lane)
 and it also includes a refclk (100MHz) differential pair
 @param lanes The number of lanes to be configured for this bundle.
+@param opt-pins Optional pin support where the value is an enum from jsl/bundles/pcie/PCIePins
 <DOC>
 
 ; for the future
 ; P/N swapping is ok per pair
 ; all lanes being swapped is ok as well
 ; this should be handled at the component level
 
-public pcb-bundle pcie (lanes:Int) :
+public pcb-bundle pcie (lanes:Int -- opt-pins:Collection<PCIePins>|False = false) :
```

Comment:
Default arguments in `pcb-bundle` definitions don't work. 

Can we please follow the pattern we have established in other functions - for example: 

```
public defn smbus (pins:SMBUSPins ...) :
  smbus-b(pins)
```

Where we create a separate function that wraps the bundle definition and allows the user to just use `smbus()` when there are no options needed.

Reply:
This comment applies to all bundles that use the optional pins.

Reply:
I tried the default argument in the pcie bundle by running the pcie example. What aspect of the default argument should not work?

Reply:
I will modify the bundle to match.

Reply:
You pass an argument in. 
Have you tried running it without an `opt-pins` argument ?

Reply:
Yes, I did. The pcie example code did both (with and without the opt-pins argument). Seemed to work.

Reply:
I'll have to check again. I've structured all the others this way because I ran into issues using the advanced keyword arguments and defaults on `pcb-bundle`. 

I wonder if it was the variadic that was not working, not the keyword args.